builtin/game/secure_sandbox.lua

```diff
@@ -0,0 +1,34 @@
+
+--[[
+This file contains things for making the global env sandbox secure that weren't
+done in C++.
+]]
+
+-- Overwrite debug.getmetatable and debug.setmetatable, so that they have the same
+-- semantics with the metatable field "__metatable_debug" as getmetatable and
```

Comment:
Why not respect the `__metatable` field?

Reply:
To give more control. And theoretically there could be a mod that does some hacky, but valid, stuff with another mod's object.

If we want to keep `debug.setmetatable` just for non-table objects and nothing else, using `__metatable` might be better.